product-service/package.json

```diff
@@ -14,10 +14,15 @@
     "@middy/http-json-body-parser": "^1.5.2",
     "@types/isomorphic-fetch": "0.0.35",
     "@types/pg": "^8.6.1",
+    "asynckit": "^0.4.0",
     "aws-sdk": "^2.989.0",
+    "combined-stream": "^1.0.8",
+    "delayed-stream": "^1.0.0",
+    "fs-extra": "^10.0.0",
     "isomorphic-fetch": "^3.0.0",
     "pg": "^8.7.1",
-    "source-map-support": "^0.5.19"
+    "source-map-support": "^0.5.19",
+    "superagent": "^6.1.0"
```

Comment:
I didnt find in your code added packages. Despite webpack should not put them to resulting bundle, unnecessary packages make noise in your repo.